spec/requests/api/api/v0/pings_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Api::V0::PingsController, type: :request do
+  describe 'GET /v0/pings' do
+    it 'should return Pong' do
+      get '/api/v0/pings'
+
+      json_response = JSON.parse(response.body)
```

Comment:
We should have that as a separate method to call on. We will be using it extensively from now on. Check the `being Stockholm API`. `spec/support/json_response.rb`.

Reply:
imo premature optimization, but ok, the things i do for love !